apis/core/v1alpha1/featureflagconfiguration_types.go

```diff
@@ -110,7 +108,7 @@ func init() {
 	SchemeBuilder.Register(&FeatureFlagConfiguration{}, &FeatureFlagConfigurationList{})
 }
 
-func GetFfReference(ff *FeatureFlagConfiguration) metav1.OwnerReference {
+func (ff *FeatureFlagConfiguration) GetReference() metav1.OwnerReference {
```

Comment:
I guess this was just a style/consistency choice since we added receivers to other functions here?

I'm still relatively new to go. If there was more of a reason for this, let me know!

Reply:
Basically yes, only a style/consistency adaptation, but I was also referring to SOLID principles here. Basically the FeatureFlagConfiguration object has all the information needed to get the owner references. Therefore I found it as a good idea to define a simple getter to retrieve this information from the object.

But generally yes, in this case it's really a choice of styling/consistency and does not have any behavioral effect on the code.

Reply:
There is also another benefit to this change. Before, a pointer to a FFConf struct was expected as input. This allows calling the function with a nil argument triggering an invalid memory access. Now, you need to have a valid FFConf struct before you can call the function